test/active_storage_validations_test.rb

```diff
@@ -271,6 +271,13 @@ class ActiveStorageValidations::Test < ActiveSupport::TestCase
     e.proc_image.attach(pdf_file)
     assert !e.valid?
     assert e.errors.full_messages.include?("Image has an invalid content type")
+
+    e = OnlyImage.new
+    e.image.attach(image_1920x1080_file)
+    e.proc_image.attach(image_1920x1080_file)
+    e.another_image.attach(tar_file_with_image_content_type)
```

Comment:
I think we should add a negative test here as well, ensuring that a `processable_image: false` will accept the .tar file as png as suggested by file extension and type.